index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
This seems a little odd. What's the use case for consuming one set of buffered metrics, and observing another? Can we simplify this down a boolean? E.g. subscribe to "mark" and buffered: true also returns all same marks.